### PR TITLE
Eelis

### PR DESCRIPTION
**Pull request by mushioov**

### CODE REVIEW

File: client/components/add-beer-view/add-beer-view.jsx [hunk 1]
```diff
@@ -107,8 +107,11 @@ class AddBeerView extends React.Component {
       bar !== DEFAULT_BAR
       && beerType !== DEFAULT_BEER_TYPE
       && volume !== DEFAULT_VOLUME
-      && abv > 0
-      && price > 0);
+      && abv > 0 && abv <= MAX_ABV
+      && price > 0) && price <= MAX_PRICE;
+    
+    const abvHint = abv >= 12 ? true : false;
+    const priceHint = price >= 30 ? true : false;
```
Comment (oamaok) [inline client/components/add-beer-view/add-beer-view.jsx]: just `const abvHint = abv >= 12` and `const priceHint = price >= 30` are enough

File: client/components/beer-listing/beer-listing.jsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import classNames from 'classnames/bind';
 import styles from './beer-listing.scss';
+import { range } from 'ramda'
 
 const css = classNames.bind(styles);
 
 export default function BeerListing({
-  beerType, volume, abv, price, location, personName, personId, rating, description,
+  beerType, volume, abv, price, location, personName, personId, rating, showRating = true, description,
```
Comment (oamaok) [inline client/components/beer-listing/beer-listing.jsx]: when using boolean props, it's usually better to default them to `false`. that way we can enable the flag when rendering the component just by writing `<BeerListing showRating />` without any assignment to the prop itself.

**Review by oamaok — CHANGES_REQUESTED** [review]
also run the linter once, some spacing issues seem to remain.

File: client/components/beer-listing/beer-listing.jsx (hunk 2 above)
Comment (mushioov) [reply client/components/beer-listing/beer-listing.jsx]: Good point

File: client/components/add-beer-view/add-beer-view.jsx (hunk 1 above)
Comment (mushioov) [reply client/components/add-beer-view/add-beer-view.jsx]: Ah yes

**Review by oamaok — APPROVED** [review]

**Pull request merged**